app/views/characters/_icon_view.haml

```diff
@@ -1,12 +1,15 @@
-- characters = name.characters.non_npcs.ordered unless local_assigns[:characters]
-- characters = characters.where(retired: false) unless local_assigns.fetch(:show_retired) { show_retired }
+-# locals: ( name: nil, characters: nil, show_template: false, show_user: false, hide_buttons: false, show_new_character_button: false, show_retired: show_retired() )
```

Comment:
Huh, I didn't know `show_retired: show_retired` would work. Does it need the parentheses?

Reply:
Ooh, ok!